TrafficLightsDemo/app_mingw.tcjs

```diff
@@ -1,38 +1,43 @@
 // Windows - MinGW
 let tc = TCF.define(TCF.CPP_TRANSFORM);
-tc.pocoLoc = 'D:\\poco-mingw';
 tc.sources = [
-    'platform:/resource/TrafficLightsDemo/TrafficLightComponent.emx#_rx2SkNXDEei3BJT-OqdJZA'
+    'platform:/resource/TrafficLightsDemo/TrafficLightComponent.emx#_rx2SkNXDEei3BJT-OqdJZA',
+];
+tc.prerequisites = [
+    'platform:/resource/LibHttpServer/httpServerLib.tcjs',
 ];
-tc.topCapsule = 'platform:/resource/TrafficLightsDemo/TrafficLightComponent.emx#_4gz4gNXDEei3BJT-OqdJZA';
-tc.createTargetProject = true;
-tc.targetProject = '/TrafficLightsDemo_target';
 tc.compilationMakeCommand = 'mingw32-make';
 
 tc.compilationMakeType = MakeType.GNU_make;
-tc.targetConfiguration = 'WinT.x64-MinGw-8.1.0';
-tc.compileArguments = '$(DEBUG_TAG) -fexceptions';
+tc.compileArguments = '$(DEBUG_TAG) -fexceptions  -DPOCO_STATIC';
+tc.compileCommand = '$(CC) ';
```

Comment:
Was it really necessary to add a space here? The default value of the "compileCommand" property is '$(CC)', did it not work with that value?

Reply:
This space is added unintentionally. This tcjs was working fine with space so I did not notice it. I will remove and commit the changes again.